apps/www/components/Questionnaire/Submissions/QuestionFeatured.js

```diff
@@ -237,7 +249,7 @@ export const QuestionFeatured = ({
       ref={questionRef}
       id={questions[0].id}
       style={{
-        padding: '0 0 46px 0',
+        padding: '30px 0 46px 0',
```

Comment:
Thanks for the fix 😄 But why not use the same padding as on the bottom?

Reply:
i don't know...